controllers/ControllerAdmin.php

```diff
@@ -116,50 +121,50 @@ public function avatar(){
 		$fileName = $_FILES['avatar']['name'];
 		$extension = ['jpg', 'jpeg', 'gif', 'png'];
 
+		$key = (new Session)->getKey('user');
+
 		if($_FILES['avatar']['error'] > 0){
 			$error =true;
-			echo $this->twig->render('profile.twig', ['error' => $error, 'logged'=> self::LOGGED, 'user'=> $_SESSION['user']['pseudo'], 'value'=> $_SESSION['user']['email']]);
+			echo $this->twig->render('profile.twig', ['error' => $error, 'logged'=> self::LOGGED, 'user'=> $key['pseudo'], 'value'=> $key['email']]);
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)